### ISSUE TITLE
How to send patches from OSX

### ISSUE THREAD

**Issue by GaryHeayes** [raspy135/Qun-synthesizer#25]
I have the Qun connected using BLE MIDI in OSX, and I can select Qun as the MIDI output device in an app called MIDIPlayerX. I've opened a MID file (one of Greg Cashin's files) and I try to 'play' it to Qun. One of two things happens: either Qun makes a 'pshhht' noise and shuts off, or it does nothing.  Any advice?  What mode should Qun be in to receive the file? (By the way, I tried connecting using IAC and USB-MIDI, but had not idea what to do, and gave up. Any advice in that area would also be welcome..).

**Comment by raspy135** [comment]
I'm not sure what kind of MIDI signal was sent to the device. Does BLE MIDI work when you use it through any DAW?
I will try MIDIPlayerX when I have time.

**Comment by GaryHeayes** [comment]
Hi, thanks for your reply. I hadn't tried though the DAW since I was using Garageband and there's no MIDI out. So I downloaded Tracktion Waveform and checked the BLE MIDI connection from there.  Once I got it connected there, I was able to send the patches from both Waveform and from MIDIPlayerX.  MIDIPlayerX is much simpler to use, but it seems to have an unfortunate side-effect of sometimes turning the volume of the Qun right down. Eg when I tried to load Greg's Mellotron patch, the volume dropped and I couldn't hear the playback. Some of the patches had the same effect, and others were fine.  When the volume dropped, I used Waveform to get it back up again, since the Qun volume level in Mix and VCF didn't seem to be able to bring it up for some reason.

So, it's working now. I got the patches loaded and they sound great (thanks Greg!). Thanks for your help.
